dev-db/mongodb/files/mongodb-5.0.26-scons.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/SConstruct b/SConstruct
```

Comment:
Please add a link to the upstream bug and also the Gentoo bug at the top of the patch. You may want to include my explanation from https://github.com/gentoo/gentoo/pull/38444#issuecomment-2533699514 too.

See also https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions.